shard-operator/src/main/java/com/redhat/service/smartevents/shard/operator/BridgeExecutorServiceImpl.java

```diff
@@ -42,7 +42,7 @@ public class BridgeExecutorServiceImpl implements BridgeExecutorService {
     public static final String KAFKA_ERROR_STRATEGY_IGNORE = "ignore";
     public static final String KAFKA_ERROR_STRATEGY_DLQ = "dead-letter-queue";
 
-    @ConfigProperty(name = "event-bridge.executor.image")
```

Comment:
no need to change this line. we already have the default value specified in `application.properties` and you can keep `event-bridge.executor.image` as property name: if there is an env variable called `SMART_EVENTS_EXECUTOR_IMAGE` the smallrye lib will inject that value by magic.

Reply:
@r00ta 
AFAIK dot(`.`) are converted to underscore(`_`) but I am not sure about hyphen(`-`).
Does they converted into underscore as well?

I also think `SMART_EVENTS_EXECUTOR_IMAGE` is making more sense then `EVENT_BRIDGE_EXECUTOR_IMAGE`. wdyt?

Reply:
@vaibhavjainwiz Just to be clear, the environment property currently supported is `EVENT_BRIDGE_EXECUTOR_IMAGE` . I would suggest we keep that one for now rather than renaming a single property with the `SMART_EVENTS_` prefix.

Reply:
I am resetting it to `EVENT_BRIDGE_EXECUTOR_IMAGE`

Reply:
done